sycl/include/sycl/property_list.hpp

```diff
@@ -64,6 +64,8 @@ class property_list : protected detail::PropertyListBase {
 
   template <typename... T> operator ext::oneapi::accessor_property_list<T...>();
 
+  using PropertyListBase::checkPropsAndThrow;
```

Comment:
This should not be exposed publicly. We can have a detail access function/class for it, but if we expose it as a public member we run the risk of users using it.

Reply:
done https://github.com/intel/llvm/pull/15253/commits/6e673acec03b839cb098117f38f8baa6ae25f49f